ddtrace/appsec/processor.py

```diff
@@ -188,8 +204,12 @@ def on_span_finish(self, span):
                 data[_Addresses.SERVER_RESPONSE_HEADERS_NO_COOKIES] = _no_cookies(response_headers)
 
         log.debug("[DDAS-001-00] Executing AppSec In-App WAF with parameters: %s", data)
-        res = self._ddwaf.run(data)  # res is a serialized json
+        res = self._ddwaf.run(data, self._waf_timeout)  # res is a serialized json
         if res is not None:
+            allowed = self._rate_limiter.is_allowed(span.start_ns)
+            if not allowed:
+                statsd.increment("_dd.py.appsec.rate_limit.dropped_traces")
```

Comment:
you might want to take the dogstatsd client that the `Tracer` configures.

you will also want to use the `datadog.tracer.*` prefix, otherwise this will count towards custom metrics and the user will get billed for it.

Reply:
RE(client): ack 🙇 
RE(metric name):Good catch! I have asked for what the impact is in the RFC. Also, a comment in the RFC marks that as a nice to have, so I can probably drop it for now.